Installation/cmake/modules/CGAL_Common.cmake

```diff
@@ -1,5 +1,10 @@
 include(${CMAKE_CURRENT_LIST_DIR}/CGAL_Macros.cmake)
 
+option(CGAL_DEV_MODE "
+Activate the CGAL developers mode. \
+See https://github.com/CGAL/cgal/wiki/CGAL_DEV_MODE"
+  FALSE)
```

Comment:
CMake does not like that multi-line documentation line.

Reply:
It says:
```
CMake Error: Parse error in cache file /home/lrineau/Git/cgal/build/CMakeCache.txt on line 812. Offending entry: Activate the CGAL developers mode. See https://github.com/CGAL/cgal/wiki/CGAL_DEV_MODE
```

Reply:
... but only the second time, when the `CMakeCache.txt` is reloaded.

Reply:
@lrineau that's your PR. Are you expecting something from somebody or you are just listing the issue?

Reply:
Nothing. I wrote those messages for myself, because yesterday I understood the issue but lacked time to write/test/commit/push the trivial fix. I was kind of in a hurry because of a personal deadline.

Reply:
Sorry for not being explicit. I will try to remember that I should explicit who is the destination of a message.